packages/sql-mapper/test/entity.test.js

```diff
@@ -267,6 +267,29 @@ test('[SQLite] throws if PK is not INTEGER', { skip: !isSQLite }, async ({ fail,
   }
 })
 
+test('[SQLite] throws if PK is not VARCHAR(255)', { skip: !isSQLite }, async ({ fail, equal, teardown, rejects }) => {
```

Comment:
You added a varchar to valid types. You should test that sqlite allows to use it. Something like that.
```js
test('[SQLite] allows to have VARCHAR PK', { skip: !isSQLite }, async ({ same, teardown }) => {
  async function onDatabaseLoad (db, sql) {
    await clear(db, sql)
    teardown(() => db.dispose())

    try {
      await db.query(sql`CREATE TABLE pages (
        id varchar(255) PRIMARY KEY,
        title varchar(255) NOT NULL
      );`)
    } catch (error) {
      console.log(error)
    }
  }
  const mapper = await connect({
    connectionString: connInfo.connectionString,
    log: fakeLogger,
    onDatabaseLoad,
    ignore: {},
    hooks: {}
  })
  const pageEntity = mapper.entities.page
  const [newPage] = await pageEntity.insert({
    fields: ['id', 'title'],
    inputs: [{ id: 'varchar_id', title: '13th page with explicit id equal to 13' }]
  })
  same(newPage, {
    id: 'varchar_id',
    title: '13th page with explicit id equal to 13'
  })
})

```

Reply:
oh i see i the prior test i had written wasn't trying testing the insertion. thank you @ivan-tymoshenko. I have adjusted as you suggested, please review